src/main/java/ml/treecaptcha/uwuify/spigot/commands/UwUCommands.java

```diff
@@ -0,0 +1,72 @@
+package ml.treecaptcha.uwuify.spigot.commands;
+
+import ml.treecaptcha.uwuify.spigot.KeyHolder;
+import ml.treecaptcha.uwuify.spigot.Uwuify;
+import org.bukkit.NamespacedKey;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.command.TabCompleter;
+import org.bukkit.entity.Player;
+import org.bukkit.persistence.PersistentDataType;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.List;
+
+public class UwUCommands implements CommandExecutor, TabCompleter {
+
+    @Override
+    public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command, @NotNull String s, @NotNull String[] strings) {
+        if(Uwuify.ALLOW_TOGGLE && sender instanceof Player p) {
+            if(strings.length == 0) {
+                return togglePlayerUwUify(p, Uwuify.UWUIFY_KEY);
+            } else {
+                switch (strings[0]) {
+                    case "chat" -> {
+                        return togglePlayerUwUify(p, KeyHolder.UWUIFY_CHAT);
+                    }
+                    case "books" -> {
+                        return togglePlayerUwUify(p, KeyHolder.UWUIFY_BOOKS);
+                    }
+                    case "signs" -> {
+                        return togglePlayerUwUify(p, KeyHolder.UWUIFY_SIGNS);
+                    }
+                    case "animal_names" -> {
+                        return togglePlayerUwUify(p, KeyHolder.UWUIFY_ANIMALS);
+                    }
+                    case "join_message" -> {
+                        return togglePlayerUwUify(p, KeyHolder.UWUIFY_JOIN_MESSAGES);
+                    }
+                    case "player_name" -> {
+                        return togglePlayerUwUify(p, KeyHolder.UWUIFY_PLAYER_NAME);
+                    }
+                    case "all" -> {
+                        return togglePlayerUwUify(p, Uwuify.UWUIFY_KEY);
+                    }
+                }
+            }
+            return false;
+        }
+        return false;
+    }
+
+    public static boolean isEnabled(Player p, NamespacedKey key) {
+        return !Boolean.parseBoolean(p.getPersistentDataContainer().getOrDefault(Uwuify.UWUIFY_KEY, PersistentDataType.STRING, "true")) && Boolean.parseBoolean(p.getPersistentDataContainer().getOrDefault(key, PersistentDataType.STRING, "true"));
```

Comment:
is there a reason for it to be a string and not a boolean?

Reply:
Besides my own stupidity, no. Will work on creating a Boolean PersistentDataType now :)

Reply:
@Roosterseatyou I think I saw byte available we could do 0x00 is false and 0x01 is true

Reply:
👍